src/readme.rs

```diff
@@ -1,82 +1,85 @@
 use aqora_config::ReadMe;
```

Comment:
I'm not sure if these updates are 100% correct. You can pass a README inline directly into the pyproject.toml. If you want to explicitly not allow those types of README, you have to throw a relevant error because its not very transparent here. I think for 100% correctness, we would have 2 functions. One for reading the README like we had before, and one for writing the README, where we either get the path and write to the file or we use `toml_edit` to write to `[readme.text]`